index.d.ts

```diff
@@ -1,294 +1,276 @@
+    timegridDisplayPrimayTime?: (time: ITimeGridHourLabel) => string;
+    timegridDisplayPrimaryTime?: (time: ITimeGridHourLabel) => string;
+    timegridDisplayTime?: (time: ITimeGridHourLabel) => string;
+    timegridCurrentTime?: (hourMarker: ITimezoneHourMarker) => string;
+    popupIsAllDay?: () => string;
+    popupStateFree?: () => string;
+    popupStateBusy?: () => string;
+    titlePlaceholder?: () => string;
+    locationPlaceholder?: () => string;
+    startDatePlaceholder?: () => string;
+    endDatePlaceholder?: () => string;
+    popupSave?: () => string;
+    popupUpdate?: () => string;
+    popupDetailDate?: (isAllDay: boolean, start: DateType, end: DateType) => string;
+    popupDetailLocation?: (schedule: ISchedule) => string;
+    popupDetailUser?: (schedule: ISchedule) => string;
+    popupDetailState?: (schedule: ISchedule) => string;
+    popupDetailRepeat?: (schedule: ISchedule) => string;
+    popupDetailBody?: (schedule: ISchedule) => string;
+    popupEdit?: () => string;
+    popupDelete?: () => string;
+}
 
-    interface IWeekOptions {
-        startDayOfWeek?: number;
-        daynames?: string[];
-        narrowWeekend?: boolean;
-        workweek?: boolean;
-        showTimezoneCollapseButton?: boolean;
-        timezoneCollapsed?: boolean;
-        hourStart?: number;
-        hourEnd?: number;
-    }
+export interface IWeekOptions {
+    startDayOfWeek?: number;
+    daynames?: string[];
+    narrowWeekend?: boolean;
+    workweek?: boolean;
+    showTimezoneCollapseButton?: boolean;
+    timezonesCollapsed?: boolean;
+    hourStart?: number;
+    hourEnd?: number;
+}
 
-    interface IMonthOptions {
-        daynames?: string[];
-        startDayOfWeek?: number;
-        narrowWeekend?: boolean;
-        visibleWeeksCount?: number;
-        isAlways6Week?: boolean;
-        workweek?: boolean;
-        visibleScheduleCount?: number;
-        moreLayerSize?: {
-            width?: string | null;
-            height?: string | null;
-        };
-        grid?: {
-            header?: {
-                height?: number;
-            },
-            footer?: {
-                height?: number;
-            }
-        };
-        scheduleFilter?: (schedule: ISchedule) => boolean;
-    }
+export interface IMonthOptions {
+    daynames?: string[];
+    startDayOfWeek?: number;
+    narrowWeekend?: boolean;
+    visibleWeeksCount?: number;
+    isAlways6Week?: boolean;
+    workweek?: boolean;
+    visibleScheduleCount?: number;
+    moreLayerSize?: {
+        width?: string | null;
+        height?: string | null;
+    };
+    grid?: {
+        header?: {
+            height?: number;
+        },
+        footer?: {
+            height?: number;
+        }
+    };
+    scheduleFilter?: (schedule: ISchedule) => boolean;
+}
 
-    interface IRaw {
-        [propName: string]: string | number | boolean | object | null;
-    }
+export interface IRaw {
+    [propName: string]: string | number | boolean | object | null;
```

Comment:
Let's use [Record<K, T>](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkt) in v2.

Reply:
👍